
message WallmonMessage
{
	required string key = 1;
	required bytes data = 2;
	
	optional double timestampMsec = 3;
	optional string hostname = 4;
	optional double sampleFrequencyMsec = 5;
	optional double sampleTimeMsec = 6;
	optional double scheduleDriftMsec = 7;
	optional uint32 networkMessageSizeBytes = 8;
}